playbooks/luna_demo_environment.yml

```
@@ -0,0 +1,81 @@
+# Manually verify your hosts has FQDN set correctly
+# check it's in /etc/hosts /etc/hostname and hostname returns it, /etc/hosts first line should be 192.168.73.1  $fqdn
+
+# TODO - add role for setting up the machine - autologin in gdm, perhaps background image etc, installation of virt-manager, vim, htop
+# TODO - refactor katello_provisioning so that it does not run installer, it should only seed stuff, installer should be customized from playbook using foreman_installer's role variables
+
+- hosts: all
+  become: true
+  roles:
+    - disable_ipv6
+    - disable_firewall
+    - selinux
+    - etc_hosts
+    - epel_repositories
+    - ostree_repositories
+    - puppet_repositories
+    - foreman_repositories
+    - katello_repositories
+    - foreman_installer
+    - libvirt
+    - katello_provisioning
+    - plugins/foreman_templates
+# openscap manual steps
+# 1. edit Base/Katello CentOS 7/OpenSCAP host group and set the policy manually
+# covered by http://projects.theforeman.org/issues/22481
+    - plugins/foreman_openscap
+# discovery manual steps (to make discovery work via pxelinux):
+# 1. unlock PXELinux global default
+# 2. change ONTIMEOUT local to ONTIMEOUT discovery
+# 3. save
+# 4. click Build PXE Default button
+# covered by http://projects.theforeman.org/issues/21204/
+    - plugins/foreman_discovery
+    - plugins/foreman_bootdisk
+    - plugins/foreman_remote_execution
+    - plugins/foreman_ansible 
+  vars:
+    katello_provisioning_organization: Demo
+    foreman_repositories_version: 1.22
+    katello_repositories_version: 3.12
+    katello_provisioning_location: Belgium
+    foreman_installer_scenario: katello
+    foreman_installer_options_internal_use_only:
+      - "--disable-system-checks"
+    foreman_installer_additional_packages:
+      - rubygem-ruby-libvirt
+      - foreman-libvirt
+      - foreman-ec2
+      - foreman-gce
+      - foreman-openstack
+      - foreman-ovirt
+      - foreman-rackspace
+      - foreman-vmware
+      - katello
```

While we wait for hammer_cli_foreman 0.17.1 to be published to the 1.22 repo, the following should work for testing:

```suggestion
      - katello
      - http://koji.katello.org/kojifiles/work/tasks/4129/214129/tfm-rubygem-hammer_cli_foreman-0.17.1-1.el7.noarch.rpm
```